packages/protoplugin/src/create-es-plugin.ts

```diff
@@ -291,14 +300,20 @@ function parseParameter(
         }
         break;
     }
+    if (printToFile) {
+      renderParams.push(`${key}=${value}`);
```

Comment:
What if my custom plugin has an option that is a simple flag, not a key-value pair?

Consider:

```yaml
opt: foo,bar=,baz=abc
``` 

I think the return type of splitParameter has to change, so that we can distinguish between `foo` - a flag, and `bar=`, a key-value with an empty value.

Might be most straight-forward to return:

```ts
{ key: string; value: string; raw: string }[]
``` 

With `raw` being the unparsed element, that we can easily join again.